### PR TITLE
[TheAnarchistLibrary.org] New Ruleset

### PR DESCRIPTION
**Pull request by chris-barry**
All the sites from this domain mentioned on their home page (recently started using Let's Encrypt)

### CODE REVIEW

**Comment by J0WI** [comment]
Thanks for you contribution!
Could you please also add comments for the two hosts that serve a mismatching cert?
sr.theanarchistlibrary.org
mk.theanarchistlibrary.org

**Comment by J0WI** [comment]
@chris-barry would be great if we could finish this one :)

**Comment by chris-barry** [comment]
Just pushed, sorry for the delay!

File: src/chrome/content/rules/TheAnarchistLibrary.org.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+	Problematic subdomains:
+	 
+	- sk.theanarchistlibrary.org *
```
Comment (J0WI) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: Typo: sk. does not exist, it's sr.

Comment (chris-barry) [inline src/chrome/content/rules/TheAnarchistLibrary.org.xml]: Good eye!

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**